bin/seed.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+
+const User = require('../models/user.model')
+
+const bcrypt = require("bcrypt")
+const bcryptSalt = 10
+
+const dbName = 'rolesPassport';
+mongoose.connect(`mongodb://localhost/${dbName}`, { useNewUrlParser: true, useUnifiedTopology: true })
+
+const boss = [{
+    username : "Juan",
+    name: "Juan",
+    password: "Pestana",
```

Comment:
```suggestion
    password: bcrypt.hashSync("Pestana", salt),
```
Te faltó pasarle _hasheada_ la password para poder logearte.